cpp/data/eigerfan/src/eigerfan_main.cpp

```diff
@@ -149,7 +149,7 @@ int parse_arguments(int argc, char** argv, EigerFanConfig &cfg)
       ("logconfig,l", po::value<std::string>(),
           "Set the log4cxx logging configuration file")
       ("threads,t", po::value<unsigned int>()->default_value(EigerFanDefaults::DEFAULT_NUM_THREADS),
-          "Set the number of 0MQ threads for the 0MQ context")
+          "Set the number of zmq threads for the zmq context")
```

Comment:
Perhaps "number of threads to create to pull detector data".

Reply:
Agreed - change made